src/js/va-letters/containers/ConfirmAddress.jsx

```diff
@@ -1,19 +1,79 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
+import { updateAddressConfirmation } from '../actions';
+
+import ProgressButton from '../../common/components/form-elements/ProgressButton';
+
 class ConfirmAddress extends React.Component {
+  constructor(props) {
+    super(props);
+    this.confirmAddress = this.confirmAddress.bind(this);
+  }
+
+  confirmAddress() {
+    this.props.onConfirmationOfAddress(true);
+    this.props.router.push('/download-letters');
+  }
+
   render() {
+    const buttons = this.props.letters.addressConfirmed !== false ?
```

Comment:
Yes I think so, it just means that these buttons will look like radio buttons instead :) so I think we'd need designers to weigh in on that.